lib/resty/timerng/thread/super.lua

```diff
@@ -18,6 +18,8 @@ local CONSTANTS_SCALING_RECORD_INTERVAL =
     require("resty.timerng.constants").SCALING_RECORD_INTERVAL
 local CONSTANTS_TOLERANCE_OF_GRACEFUL_SHUTDOWN =
     require("resty.timerng.constants").TOLERANCE_OF_GRACEFUL_SHUTDOWN
+local CONSTANTS_SCALING_LOG_INTERVAL =
+    require("resty.timerng.constants").SACALING_INFO_LOG_INTERVAL
```

Comment:
We should use one line `require("resty.timerng.constants")` to reduce duplicated code.

Reply:
done